src/ontology/components/mondo-subsets.owl

```diff
@@ -45774,7 +45756,7 @@
         <owl:annotatedSource rdf:resource="http://purl.obolibrary.org/obo/MONDO_0008271"/>
         <owl:annotatedProperty rdf:resource="http://www.geneontology.org/formats/oboInOwl#hasDbXref"/>
         <owl:annotatedTarget>GARD:2256</owl:annotatedTarget>
-        <oboInOwl:source rdf:datatype="http://www.w3.org/2001/XMLSchema#string">Orphanet:93337</oboInOwl:source>
+        <oboInOwl:source>Orphanet:93337</oboInOwl:source>
```

Comment:
Why is the `rdf:datatype` removed here vs. added in [line 761](https://github.com/monarch-initiative/mondo/pull/7392/files#diff-7840a3d5c21db9ff4a6fd767519063766c2d3ad5fee232689bfc287f43c979caL761)?

Reply:
@matentzn are these minor differences^^ anything we should look into further for possible updates down the line?

Reply:
No problem, you can merge it. The way to fix it for @twhetzel if she wants is to update https://github.com/monarch-initiative/mondo/blob/35bebd4527781de6350fdfcc7f0df1ec0fcd8fc6/.github/workflows/diff.yaml#L11